### DISCUSSION TITLE
Add Categories to Prompt Suggestions (Like in ChatGPT)

### DISCUSSION THREAD

**Discussion by frettig-radiox** [open-webui/open-webui#6294] [Ideas]
### **Feature Request: Add Categories to Prompt Suggestions**

I would like to propose the following feature:

The input forms for **"Default Prompt Suggestions"** (Admin Panel -> Interface) and **"Prompt Suggestions"** (Workspace -> Models -> Edit) should be unified, and a new input field for **Category** should be added.

![Bild1](https://github.com/user-attachments/assets/63941d49-fb36-43b1-a528-1e80942be496)

Proposed GUI:

- Title  
- Subtitle  
- Prompt  
- Category (this would be new)

**Behavior:**

This feature would be particularly useful when managing a large number of prompt suggestions. If multiple categories are assigned, the user will initially be offered a category selection (similar to ChatGPT), improving organization and navigation. If all prompts belong to the same category, the current behavior will remain unchanged, and all prompt suggestions will be displayed directly.

This ensures that in cases with few prompts, the simplicity and clarity of the GUI are maintained, without adding unnecessary complexity.

![Bild2](https://github.com/user-attachments/assets/37e6c0a4-0d44-430d-8f73-f45ac15050e5)

**Comment by flefevre** [comment]
I totally agree with you, We need to find a solution to better managed prompts over open webui.
Adding category? Adding job category?

Perhaps also to give access to workflow of prompt: by clicking it give you access to a first prompt step, you execute , you interact, when happy you jump to the second prompt step etc... Until you reach the end of the pipeline/workflow dedicated to a specific huge tasks.


To ease the use for non expert.

It could be useful to group them in another tab to manage the 'prompt life cycle' with different version, highlights of parameters etc ..

Perhaps prompts could be also linked to 'team features. A team has access to specific prompt version.

It could be good to connect to other prompt management system such as langfuse  https://github.com/open-webui/open-webui/discussions/5915 or build a dedicated module in openwebui.
